CMakeLists.txt

```diff
@@ -14,12 +14,17 @@
 # limitations under the License.
 #===============================================================================
 
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.18.2)
 
 if(POLICY CMP0022)
     cmake_policy(SET CMP0022 NEW)
 endif()
 
+# Compiler id for Apple Clang is now AppleClang
+if(POLICY CMP0025)
+    cmake_policy(SET CMP0025 NEW)
+endif()
+
 # Foo::Bar always refers to an IMPORTED target
 if(POLICY CMP0028)
```

Comment:
(minor) Some of the policies might not be needed anymore with CMake version update.